src/init.cpp

```diff
@@ -748,7 +754,6 @@ bool AppInit2(boost::thread_group& threadGroup)
 #endif // ENABLE_WALLET
 
     fIsBareMultisigStd = GetArg("-permitbaremultisig", true) != 0;
```

Comment:
This should probably be moved as well. The help message is currently in "connection options": https://github.com/jtimon/bitcoin/blob/d66803ee06175d71ff7605ed1cfa8b91aec23f62/src/init.cpp#L284

Very exciting branch by the way! :)